diff --git a/config.def.h b/config.def.h
index 877afab..b077482 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+char font[] = "Fira Mono:size=10:antialias=true:autohint=true";
 int borderpx = 2;
 
 /*
@@ -32,7 +32,7 @@ float chscale = 1.0;
  *
  * More advanced example: " `'\"()[]{}"
  */
-static char worddelimiters[] = " ";
+static char worddelimiters[] = " `'\"()[]{}";
 
 /* selection timeouts (in milliseconds) */
 unsigned int doubleclicktimeout = 300;
@@ -85,30 +85,28 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	[0] = "#212121", /* black   */
+	[1] = "#c30771", /* red     */
+	[2] = "#10a778", /* green   */
+	[3] = "#a89c14", /* yellow  */
+	[4] = "#008ec4", /* blue    */
+	[5] = "#523c79", /* magenta */
+	[6] = "#20a5ba", /* cyan    */
+	[7] = "#d9d9d9", /* white   */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	[8]  = "#424242", /* black   */
+	[9]  = "#fb007a", /* red     */
+	[10] = "#5fd7af", /* green   */
+	[11] = "#f3e430", /* yellow  */
+	[12] = "#20bbfc", /* blue    */
+	[13] = "#6855de", /* magenta */
+	[14] = "#4fb8cc", /* cyan    */
+	[15] = "#f1f1f1", /* white   */
+
+	/* special colors */
+	[256] = "#212121", /* background */
+	[257] = "#d9d9d9", /* foreground */
 };
 
 
@@ -118,8 +116,8 @@ const char *colorname[] = {
  */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
-unsigned int defaultcs = 256;
-unsigned int defaultrcs = 257;
+unsigned int defaultcs = 12;
+unsigned int defaultrcs = 0;
 
 /*
  * Default shape of cursor
